css/.scss-lint.yml

```diff
@@ -12,6 +12,11 @@ linters:
         space_before_bang: true
         space_after_bang: false
 
+    # Checks for BEM selectors with more elements than a specified maximum number.
+    BemDepth:
+        enabled: false
```

Comment:
Sure, I ll turn that on. So if we write `.block__element__subelement` and we will get error.